docs/regional_coverage/statistics/energy-balance.md

```diff
@@ -2,20 +2,17 @@
 
 General Information 
 
-An energy balance provides a detailed and comprehensive overview of the energy situation in a specific country or region. Several pieces of information are contained:
+An energy balance provides a detailed and comprehensive overview of the energy situation in a specific country or region. The energy balance provides all relevant information to describe all end-use sectors (agriculture, commercial, industry, residential and transport) for an energy system model. Therefore, energy carriers, such as coal, oil and gas are covered in the energy statistics flow. Several pieces of information are contained:
 1.	Energy Production and Sources: It shows the total energy produced within a country (or region), including the breakdown of different energy sources (e.g. oil, natural gas, coal, renewables, and nuclear power). This helps in understanding a country's energy mix and its dependence on specific energy sources.
 2.	Energy Imports and Exports: The balance details the amounts of energy imported and exported. This information is crucial for understanding a country's energy dependency on other nations and its role in the global energy market.
 3.	Energy Transformation: It tracks how primary energy sources (e.g. crude oil or natural gas) are converted into secondary forms of energy (e.g. electricity or refined petroleum products) that are more usable for end-use sectors.
 4.	Energy Consumption: The balance provides data on the total energy consumption, as well as the distribution of this consumption across different sectors such as residential, industrial, transportation, and commercial. It can also show the efficiency of energy use in these sectors.
 
-### Energy Flows
-
-The energy balance provides all relevant information to describe all end-use sectors (agriculture, commercial, industry, residential and transport) for an energy system model. Therefore, energy carriers, such as coal, oil and gas are covered in the energy statistics flow.
 Depending on the level of modelling, the energy balance does not support the degree on detail. Therefore, some further expert information is necessary in order to build up a full model.
 
 ### Energy Balance for TIAM
 
-The energy balances for TIAM are provided by the IEA. There is an open data energy balance, provided by the IEA, available that gives an overview of energy production and transformation:
+TIAM uses energy balance provided by the International Energy Agency (IEA). There is an open data energy balance that gives an overview of energy production and transformation:
 
 https://stats.oecd.org/BrandedView.aspx?oecd_bv_id=enestats-data-en&doi=data-00512-en
```

Comment:
Please use the following format to include this URL in the paragraph above [energy balance](URL)